apps/api/src/server/app/case-officer/case-officer.service.js

```diff
@@ -1,23 +1,26 @@
+import appealRepository from '../repositories/appeal.repository.js';
 import newReviewRepository from '../repositories/review-questionnaire.repository.js';
 import { transitionState } from '../state-machine/transition-state.js';
-import appealRepository from '../repositories/appeal.repository.js';
-import { buildAppealCompundStatus } from '../utils/build-appeal-compound-status.js';
 import { breakUpCompoundStatus } from '../utils/break-up-compound-status.js';
+import { buildAppealCompundStatus } from '../utils/build-appeal-compound-status.js';
 
 /** @typedef {import('@pins/api').Schema.Appeal} Appeal */
 
-const reviewComplete = function (reviewReason) {
-	return Object.keys(reviewReason).every((index) => !reviewReason[index])? true : false;
+const reviewComplete = (reviewReason) => {
+	return !!Object.keys(reviewReason).every((index) => !reviewReason[index]);
```

Comment:
I think we can probably remove the `!!` here?

Reply:
correct, done!